op-test

```diff
@@ -644,3 +644,18 @@ except Exception as e:
     optestlog.error("Exit unexpectedly with Exception={}".format(e))
     OpTestConfiguration.conf.util.cleanup()
     exit(-1)
+finally:
+    # Create a softlink to `latest` test results
+    output = OpTestConfiguration.conf.logdir
+    if not os.path.exists(output):
+        optestlog.error("Test result folder not found")
+        exit(-1)
+    basedir = os.path.abspath(os.path.join(output, os.pardir))
+    latest = os.path.join(basedir, "latest")
+    if os.path.exists(latest):
+        if os.path.islink(latest):
+            os.unlink(latest)
+        else:
```

Comment:
`else` part is not required, instead we can handle in L661,
```python
try:
    os.symlink(output, latest)
except OSError:
    optestlog.error()
```

Reply:
Sure..